htdocs/xoops_lib/Xoops/Core/Logger.php

```diff
@@ -108,27 +108,27 @@ public function handleError($errno, $errstr, $errfile, $errline)
 
             switch ($errno) {
                 case E_USER_NOTICE:
-                    $msg = (class_exists('\XoopsLocale', false) ? \XoopsLocale::E_LOGGER_ERROR : '*Error') . $msg;
+                    $msg = (class_exists('\XoopsLocale', false) ? \XoopsLocale::E_LOGGER_NOTICE : '*Notice:') . $msg;
```

Comment:
Error is appropriate, as E_USER_NOTICE is the highest possible non-fatal user error, and historically has been reported as an error.